pydantic_ai_examples/stream_whales.py

```diff
@@ -85,7 +85,7 @@ async def main():
                     table.add_row(
                         str(wid),
                         whale['name'],
-                        f'{whale['length']:0.0f}',
+                        f'{whale["length"]:0.0f}',
```

Comment:
According to pycharm, you can't nest f-strings with different quote types in python<=3.10. I think we support that so I changed this.